website/server/libs/api-v3/analyticsService.js

```diff
@@ -144,7 +144,7 @@ let _generateValueForGoogleAnalytics = (data) => {
 
 let _sendDataToGoogle = (eventType, data) => {
   let eventData = {
-    ec: data.category,
+    ec: data.category || data.gaCategory || 'behavior',
```

Comment:
Sure, good point. I did it this way for best consistency with the behavior where it only looked at `data.category`, but I'll turn it around!